src/page/VariantStore.ts

```diff
@@ -1,12 +1,13 @@
 import { observable } from 'mobx';
-import { remoteData, VariantAnnotation } from 'cbioportal-frontend-commons';
+import {
+    remoteData,
+    VariantAnnotation,
+    IndicatorQueryResp,
+} from 'cbioportal-frontend-commons';
 import client from './genomeNexusClientInstance';
 import oncokbClient from './OncokbClientInstance';
 import MobxPromise from 'mobxpromise';
-import {
-    IndicatorQueryResp,
-    Gene,
-} from 'cbioportal-frontend-commons/api/generated/OncoKbAPI';
+import { Gene } from 'cbioportal-frontend-commons/dist/api/generated/OncoKbAPI';
```

Comment:
actually you can do
 ```suggestion
import { OncoKbGene } from 'cbioportal-frontend-commons';
```

but, again, `oncokb-api-client` should be a separate package...